src/commands_player_3.c

```diff
@@ -51,8 +49,7 @@ int		command_incantation_count_player(t_zappy *var, t_player *p, int *pl)
 	{
 		if (var->players[i].status == FD_CLIENT &&
 			var->players[i].level == p->level &&
-			var->players[i].coord[0] == p->coord[0] &&
-			var->players[i].coord[0] == p->coord[0])
+			same_square(var->players[i].coord, p->coord))
 		{
 			++pl[i];
 			++nb_player;
```

Comment:
just underneath the whole else condition is useless cause a bzero is done on the array just after malloc.